accounts/views.py

```diff
@@ -1,11 +1,51 @@
 from django.http import HttpResponsePermanentRedirect
 from django.contrib.auth import logout as auth_logout
 from django.core.urlresolvers import reverse_lazy
+from django.shortcuts import render
+from django.utils.translation import ugettext as _
 from django.views.generic import RedirectView, ListView, DetailView
 
 from models import UserProfile
 
 
+def user_settings_view(request):
```

Comment:
Widzę w czym jest problem, jednak bym próbował to zrobić updateview. Pogadamy o tym we wtorek, ok?